src/transactions/feed/JumpstartTransactionDetailsScreen.tsx

```diff
@@ -90,10 +99,34 @@ function JumpstartTransactionDetailsScreen({ route }: Props) {
   const token = useTokenInfo(transaction.amount.tokenId)
   const isDeposit = transaction.type === TokenTransactionTypeV2.Sent
   const jumpstartContractAddress = transaction.address
+  const networkId = transaction.networkId
+  const tokenAmount = transaction.amount
 
   const walletAddress = useSelector(walletAddressSelector)
   const bottomSheetRef = useRef<BottomSheetRefType>(null)
 
+  const [error, setError] = React.useState<Error | null>(null)
```

Comment:
i don't think you need this state variable either? it seems like you are not using the value of this error so you can replace it with `fetchClaimData.error || reclaimStatus === 'error'`?

Reply:
speaking of which, i think the `fetchClaimData` error handler needs to be populated

Reply:
Hmm, yeah, that should work... `reclaimStatus` could be on `error` when we first load this view because of the last attempt, so I have to ignore the reclaimStatus if the users didn't request the refetch, that's why I'm using `isReclaimIniciated`.

Reply:
Actually, I need a variable for this to implement the dismiss behavior.